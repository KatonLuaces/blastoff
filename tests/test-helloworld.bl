print([65;66;67;68]);