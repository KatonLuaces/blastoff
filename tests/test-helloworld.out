
  2x3 GraphBLAS int32_t matrix, hypersparse by row
  no entries

